Source/Runtime/Conditions/Poke interaction/Readme.md

```diff
@@ -0,0 +1,17 @@
+## Poke interactions
+
+# Recommended versions
+
+newest Unity 2022
+newest VR Builder
```

Comment:
This will get obsolete so quickly. Maybe state something like "This condition was built on Unity 2022 with VR Builder 2.7.1".